content/docs/rfcs/10/README.md

```diff
@@ -267,7 +303,8 @@ There are multiple implementations of Waku v2 and its protocols:
 - [go-waku (Go)](https://github.com/status-im/go-waku/)
 - [js-waku (NodeJS and Browser)](https://github.com/status-im/js-waku/)
 
-Below you can find an overview of the specs that they implement as they relate to Waku. This includes Waku v1 specs, as they are used for bridging between the two networks.
+Below you can find an overview of the specs that they implement as they relate to Waku.
```

Comment:
```suggestion
Below you can find an overview of the specs that they implement as they relate to Waku v2.
```

In general, I suggest using "Waku v2" instead of "Waku" as much as possible.